pagefaultgames/pokerogue
# [Localization] Updated pixel font for Simplified Chinese

## What are the changes?


Introduced pixel font for Chinese, to match the game style.

The font comes from an open-source project https://github.com/TakWolf/fusion-pixel-font

And you can find the licence here https://github.com/TakWolf/ark-pixel-font/blob/develop/LICENSE-OFL

## Why am I doing these changes?




The game currently uses a font that only contains basic latin characters.

All the CJK characters are fall back to a system defined font.

## What did change?



Added a pixel font in the static resource, and use the font when the language is set to Simplified Chinese.

There're too many places which are designed for "emerald" font.

After the change, the "emerald" font still take the precedence over the newly introduced pixel font.

### Screenshots/Videos



<img width="766" alt="main" src="https://github.com/pagefaultgames/pokerogue/assets/1233893/844980bf-7c29-406e-b3b5-58d2842e7646">

<img width="768" alt="battle" src="https://github.com/pagefaultgames/pokerogue/assets/1233893/dc10fba2-ce2c-48d1-a790-c08b34241656">

## How to test the changes?





Please switch the language to Chinese Simplified to verify the change.

<img width="766" alt="change" src="https://github.com/pagefaultgames/pokerogue/assets/1233893/449cff1d-7dc2-4351-9d55-57fe3c459e80">

## Checklist
